src/app/containers/InlineLink/index.jsx

```diff
@@ -25,8 +25,12 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
   if (result) {
     // the path is the first item in the array
     const path = result[0];
+    const anchor = locator.split('#')[1] || null;
```

Comment:
I don't think anchor is the correct naming - this is a hash. Can we maybe rename to hash throughout this PR?

Reply:
Noted. Thank you.

Reply:
Done!